pkg/harvester/components/settings/auto-rotate-rke2-certs.vue

```diff
@@ -0,0 +1,85 @@
+<script>
+import CreateEditView from '@shell/mixins/create-edit-view';
+import { RadioGroup } from '@components/Form/Radio';
+import UnitInput from '@shell/components/form/UnitInput';
+
+export default {
+  name: 'HarvesterAutoRotateRKE2Certs',
+
+  components: { RadioGroup, UnitInput },
+
+  mixins: [CreateEditView],
+
+  data() {
+    let parseDefaultValue = {};
+
+    try {
+      parseDefaultValue = JSON.parse(this.value.value);
+    } catch (error) {
+      parseDefaultValue = JSON.parse(this.value.default);
+    }
+
+    return { parseDefaultValue };
+  },
+
+  created() {
+    this.update();
+  },
+
+  methods: {
+    update() {
+      const value = JSON.stringify(this.parseDefaultValue);
+
+      this.$set(this.value, 'value', value);
+    }
+  },
+
+  watch: {
+    value: {
+      handler(neu) {
+        const parseDefaultValue = JSON.parse(neu.value);
+
+        this.$set(this, 'parseDefaultValue', parseDefaultValue);
+      },
+      deep: true
+    }
+  }
+};
+</script>
+
+<template>
+  <div class="row">
+    <div class="col span-12">
+      <RadioGroup
+        v-model="parseDefaultValue.enable"
+        class="mb-20"
+        name="model"
+        :options="[true,false]"
+        :labels="[t('generic.enabled'), t('generic.disabled')]"
+        @input="update"
+      />
+      <UnitInput
+        v-if="parseDefaultValue.enable"
+        v-model="parseDefaultValue.expiringInHours"
+        v-int-number
+        class="mb-20"
+        :min="1"
+        :max="8758"
```

Comment:
[UnitInput component](https://github.com/a110605/harvester-dashboard/blob/1b7450ef2330ea05603606f26ed2754355474d31/shell/components/form/UnitInput.vue#L113) from shell doesn't have `max` prop only `min`.

User is able to input expireHour to 8759.
Backend error msg said the max number is 8759 not 8758.

<img width="1264" alt="Screenshot 2024-05-09 at 2 55 09 PM" src="https://github.com/harvester/dashboard/assets/5744158/05a549e9-1b64-4933-b5e4-5f8104e9cb61">

Reply:
The value should be `<= 8759`.

https://github.com/harvester/harvester/blob/5ab43f02144fb8c193005905945d60cc65a73133/pkg/webhook/resources/setting/validator.go#L935-L938

Reply:
@torchiaf, sorry, I wrote the wrong value in description. Updated it.

Reply:
So we can set `:max="8759"`

Reply:
Good catch! Thank you.